stache.cabal

```diff
@@ -52,7 +52,7 @@ library
         mtl >=2.1 && <3.0,
         template-haskell >=2.11 && <2.19,
         text >=1.2 && <2.1,
-        vector >=0.11 && <0.13
+        vector >=0.11 && <0.14
```

Comment:
@felixminom looks good. Could you update the package version and add an entry to the CHANGELOG, please?